src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Unix.cs

```diff
@@ -59,9 +60,9 @@ public static string GetDistroVersionString()
                 return "OSX Version=" + s_osxProductVersion.ToString();
             }
 
-            DistroInfo v = ParseOsReleaseFile();
+            DistroInfo v = GetDistroInfo();
 
-            return "Distro=" + v.Id + " VersionId=" + v.VersionId + " Pretty=" + v.PrettyName + " Version=" + v.Version;
+            return "Distro=" + v.Id + " VersionId=" + v.VersionId;
         }
 
         private static readonly Version s_osxProductVersion = GetOSXProductVersion();
```

Comment:
`GetOSXProductVersion()` should be able to be removed as well and just use `RuntimeEnvironment`.

Reply:
Sorry, I meant `GetOSXKernelVersion()`.  That is the method that appears to be duplicated with PlatformAbstractions.

`RuntimeEnvironment.OperatingSystemVersion` will give you the same value as what `GetOSXKernelVersion()` will return.

Reply:
sorry for delay - had to suspend this PR. The value was not identical since they adjust version a little bit (`10.<old major version - 4>.<rest of version>`) but there were only two usages of that so I've changed those